README.md

```diff
@@ -76,5 +76,5 @@ The frontend and backend applications are deployed to two Azure Web Applications
 1. Building new Docker images with (from the relevant frontend/backend directory) `docker build --target production --tag calpin/playlist-manager-"frontend/backend":prod .`
 2. Pushing the new image to Docker Hub `docker push calpin/playlist-manager-"frontend/backend":prod`
 3. Sending a curl request to the appropriate Webhook url:
-    - Frontend: `curl -dH -X POST "https://\$PlayMan:d17p8pbD0t6xpCdep4tPdetcbsTlZiomZ6kwAwuJB9XaXwjE5hmmgSpBTYqQ@playman.scm.azurewebsites.net/api/registry/webhook"`
-    - Backend: `curl -dH -X POST "https://\$PlayManBackend:aBPc2NHMrYLEHQtmPB1JhQKgtsonxezPjliwxptyx2142STiv91aRxmnmKe6@playmanbackend.scm.azurewebsites.net/api/registry/webhook"`
+    - Frontend: `curl -dH -X POST "Frontend Webhook from Azure"`
```

Comment:
Ah, caught! But I think not rotated in Azure still!

Reply:
Should now be rotated 😬

Reply:
Let's merge and see what happens 😀